test/onnx/test_pytorch_onnx_onnxruntime_cuda.py

```diff
@@ -65,6 +65,24 @@ def forward(self, input, target):
         target[target == 1] = -100
         self.run_test(FusionModel(), (input, target))
 
+    @skipIfNoCuda
+    @disableScriptTest()
+    def test_apex_o2(self):
```

Comment:
do we need a training test?

Reply:
no since the apex O2 hook applies for both eval and training.